src/find-nargo.ts

```diff
@@ -1,6 +1,15 @@
+import os from 'os';
+import path from 'path';
+import fs from 'fs';
 import which from 'which';
+import { NargoNotFoundError } from './noir';
+import { MarkdownString } from 'vscode';
 
-const nargoBinaries = ['nargo'];
+// List of possible nargo binaries to find on Path
+// We prioritize 'aztec-nargo' as more specialised case
+const nargoBinaries = ['aztec-nargo', 'nargo'];
+// List of possible default installations in users folder
+const nargoInstallLocationPostix = ['.aztec/bin/aztec-nargo', '.nargo/bin/nargo'];
```

Comment:
that means we would have different nargo for each project on the workspace and they potentially misalign.

On the other hand we'd have setting where you can set path to nargo so that is unfortunate default. If we do not put that as priority then I think that aztec-nargo probably won't matter in the list as I would expect most to have both nargo and aztec-nargo. Aztec-nargo is more specialized mostly for people who are not first time users?

Reply:
we can add checking for project file if it not indicates contract ad pointed in parent issue.

Reply:
ideally project that was created with specialized compiler would have that indicated as dependency in project file?

Reply:
if we have old version of another compiler there would be a warning/error on the version itself